### PR TITLE
implemented tokenSaleBox

### PR DESCRIPTION
**Pull request by SeunghunSunmoonLee**

### CODE REVIEW

File: src/components/Main/index.js [hunk 1]
```diff
@@ -89,34 +88,22 @@ const Main = () => (
...
+        <div className="token-sale-header" style={{display: 'flex', paddingLeft: '199px', alignItems: 'center', verticalAlign: 'middle'}}>
+          <div style={{verticalAlign: 'center', height: '36px', padingTop: '5px'}}>Token Distribution</div>
         </div>
-        <div className="token-sale-box-body" style={{display: 'flex', alignItems: 'center', flexDirection: 'column', width: '100%', height: '100%', overflow: 'hidden'}}>
-          <div className="graph" style={{marginTop: '45px'}}>
-            <div className="graph-left" style={{width: `${tokenSales[4].text.split('%')[0]}%`}}>
-            </div>
-            <div className="">
-            </div>
+        <img className="ares-coin" src="/images/arestech-coin.png" alt="arestech-coin" style={{position: 'absolute', width:"198", height:"198", left: '5px', top: '20px'}}/>
```
Comment (geastwood) [inline src/components/Main/index.js]: This seems to be not formatted with prettier. For example there should be a space before />

File: src/components/Main/index.js [hunk 2]
```diff
@@ -89,34 +88,22 @@ const Main = () => (
...
+        <div className="token-sale-box-body" style={{display: 'flex', paddingLeft: '199px', flexDirection: 'column', width: '100%', height: '100%', overflow: 'hidden'}}>
+          <div className="token-sale-header2" style={{ marginTop: '15px', height: '38px', fontSize: '40px', fontWeight: 'bold', color: '#001ada'}}>
+            Begins 09.2018
```
Comment (geastwood) [inline src/components/Main/index.js]: The multi-lang support is already in master, if you rebase on latest master you should be able to use it. These texts guess need to be be extracted in the `src/data/messages/en.js`. Same with the one below.

**Comment by geastwood** [comment]
Seems not optimised for mobile view. But leave you to decide.

**Comment by chutium** [comment]
This PR was automatically merged because of the commits in this PR included in another PR #18...

But the changes are reverted. Please fix mobile view and use mult-lang support for the content in this box.

**Comment by SeunghunSunmoonLee** [comment]
Mobile view should look the same as desktop design? It can't be the same since the coin image height to container height will be much smaller than before. Another challenge is font size should be reduced significantly to fit into 65% of the container with

**Pull request merged**